src/buffer/out/textBuffer.cpp

```diff
@@ -2306,9 +2311,9 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
             try
             {
                 // TODO: MSFT: 19446208 - this should just use an iterator and the inserter...
-                const auto glyph = row.GetCharRow().GlyphAt(iOldCol);
-                const auto dbcsAttr = row.GetCharRow().DbcsAttrAt(iOldCol);
-                const auto textAttr = row.GetAttrRow().GetAttrByColumn(iOldCol);
+                const auto glyph = chars->Char();

```

Comment:
Hopefully this doesn't cause any trouble with DBCS. It might be worth just smoke testing that -- fill a buffer with `U+30AB KATAKANA LETTER KA (カ)` and size it around to make sure it reflows properly.

Reply:
![gh-12637-dbcs](https://user-images.githubusercontent.com/18356694/158887378-65f6105d-4d26-4528-8fd6-01ae9cf05c86.gif)